R/timer.R

```diff
@@ -101,7 +101,7 @@ RemoveBatchEffect <- function(cancer.exp, immune.exp, immune.cellType) {
   tmp0 <- c()
   for(kk in unique(names(immune.cellType))){
     tmp.vv <- which(names(immune.cellType)==kk)
-    tmp0 <- cbind(tmp0, apply(immune.exp.br[, tmp.vv], 1, median, na.rm=T))
```

Comment:
Thanks for fixing these! These are relicts from code I've stolen elsewhere.

Reply:
No prob 😉 
I just run BiocCheck on top of the classical R CMD check - it highlights a couple of items more that I feel indeed belong to the pool of good practices.